components/question-page.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/boxicons@latest/css/boxicons.min.css" />
+    <link rel="stylesheet" href="https://algernui.netlify.app/components/main.css" />
+
+    <link rel="stylesheet" href="../styles/style.css">
+    <link rel="stylesheet" href="../styles/question-page.css">
+
+    <title>Algern Quiz</title>
+</head>
+
+<body>
+    <header class="header">
+        <nav class="navbar">
+            <a href="../index.html">
```

Comment:
you can use / only to navigate to the index.html

Reply:
done